src/main/java/org/peergos/config/IdentitySection.java

```diff
@@ -0,0 +1,36 @@
+package org.peergos.config;
+
+
+import io.libp2p.core.PeerId;
+import org.peergos.util.JsonHelper;
+
+import java.util.Base64;
+import java.util.LinkedHashMap;
+import java.util.Map;
+
+public class IdentitySection implements Jsonable {
+    public final byte[] privKey;
```

Comment:
Maybe call this privKeyProtobuf because it is protobuf?

Reply:
done